node/lib/util/open.js

```diff
@@ -216,18 +229,30 @@ Opener.prototype.isOpen = co.wrap(function *(subName) {
  * is *unset*; since this operation is expensive, we cannot do it automatically
  * each time a submodule is opened.
  *
- * @param {String} subName
+ * @param {String}  subName
+ * @param {boolean} bare
  * @return {NodeGit.Repository}
  */
-Opener.prototype.getSubrepo = co.wrap(function *(subName) {
+Opener.prototype.getSubrepo = co.wrap(function *(subName, bare=false) {
```

Comment:
another instance -- js doesn't have keyword arguments.

Reply:
this is a default parameter...
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Default_parameters